### DISCUSSION TITLE
Devices not recognized that appear to be in the DB

### DISCUSSION THREAD

**Discussion by Kilkenny-82** [Koenkk/zigbee2mqtt#7894] [Devices]
Hello all, 
Im using Zigbee2MQTT as an add-on within my Home Assistant, already added a bunch of devices, and it all worked perfectly. Recently I got 2 new devices from TuYa Zigbee that appeared to be supported, one is actually a device I had already but wanted a second one off. But both appear as unsupported devices.
First is a 4-button switch registered as TS004F, the other is an Air Quality Sensor registered as TS0601. Now I tried adding them as new devices, but then I got a whole bunch of errors about missing libraries, which I indeed cant find in my installation.
So my questions are:
How come devices I find on the Github aren't supported in my Home Assistant/Zigbee2MQTT?
f I want to add something myself, do I also need to add the Herdsman Converter files, or where are those modules located in HA?
Thanks in advance for the support

**Comment by Kilkenny-82** [comment]
Anyone that can help me a bit further?

**Reply by evangelo18** [reply]
did you ever resolve this? I am having the same issue

**Reply by Kilkenny-82** [reply]
It was solved with an update, it seemed my HA was looking at the wrong repository therefor my Zigbee2MQTT never got updated
